libs/bluegl/CMakeLists.txt

```diff
@@ -19,7 +19,11 @@ set(SRCS
 
 if (WIN32)
     set(SRCS ${SRCS} src/BlueGLWindows.cpp)
-    set(SRCS ${SRCS} src/BlueGLCoreWindowsImpl.S)
+    if(${CMAKE_SIZEOF_VOID_P} STREQUAL 4)
```

Comment:
Same here, also why is the test string based whereas above it was integer based?

Reply:
Oh... it will be my mistake.